CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.60.0]
+
+### Breaking Changes
+* Autoharness Subcommand by @carolynzech in https://github.com/model-checking/kani/pull/3874
```

Comment:
This still appears under "Breaking Changes". Please add it under a separate "Major Changes" section.